filesystem/storage/DBFile.php

```diff
@@ -1,9 +1,20 @@
 <?php
 
+namespace SilverStripe\Model\FieldType;
+
 use SilverStripe\Filesystem\ImageManipulation;
 use SilverStripe\Filesystem\Storage\AssetContainer;
 use SilverStripe\Filesystem\Storage\AssetStore;
 
+use Injector;
+use AssetField;
+use File;
+use Director;
+use Permission;
+use ShortcodeHandler;
+use ValidationResult;
+use ValidationException;
+
 // Un-comment once https://github.com/silverstripe/silverstripe-framework/pull/4551/ is merged
 // namespace SilverStripe\Filesystem\Storage;
```

Comment:
This begs the question; We ARE planning on psr-4 standardising the directories to these namespaces right?